examples/open_deep_research/scripts/text_inspector_tool.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional
+
+from smolagents import Tool
+from smolagents.models import MessageRole, Model
+
+from .mdconvert import MarkdownConverter
```

Comment:
Is this working? Feel free to ignore the suggestion.
```suggestion
from mdconvert import MarkdownConverter
```